pkg/scheduler/framework/session_plugins.go

```diff
@@ -791,21 +841,19 @@ func (ssn *Session) NodeOrderReduceFn(task *api.TaskInfo, pluginNodeScoreMap map
 // BuildVictimsPriorityQueue returns a priority queue with victims sorted by:
 // if victims has same job id, sorted by !ssn.TaskOrderFn
 // if victims has different job id, sorted by !ssn.JobOrderFn
-func (ssn *Session) BuildVictimsPriorityQueue(victims []*api.TaskInfo) *util.PriorityQueue {
+func (ssn *Session) BuildVictimsPriorityQueue(victims []*api.TaskInfo, preemptor *api.TaskInfo) *util.PriorityQueue {
```

Comment:
The `AddQueueOrderFn` has already defined the hierarchical queue's order, so it seems that VictimJobOrderFn and VictimTaskOrderFn not necessary？just define the job order and task order is ok?

Reply:
I did modify the code as suggested earlier, but during subsequent iterations, I discovered that the modification would impact the functionality. For hierarchical queues, we need to first preempt queues that belong to the same parent queue as the preemptor. This means that when sorting the victims, it is necessary to have information about the preemptor. Therefore, it is necessary to add a victimQueueOrderFn here.